test/distributed/algorithms/quantization/test_quantization.py

```diff
@@ -22,6 +22,14 @@ def _build_tensor(size, value=None, dtype=torch.float, device_id=None):
     else:
         return torch.empty(size, size, size, dtype=dtype).fill_(value).cuda(device_id)
 
+def _build_2Dtensor(size, value=None, dtype=torch.float, device_id=None):
+    if value is None:
```

Comment:
what's the value of this util function when this is exactly the same with `_build_tensor`? can't it merge with the `_build_tensor` by specifying the dimension size in the arg? In fact I think we should just treat `size` arg as the real argument to torch.empty? like `size` should be a List[int]